node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/StorageMaintainer.java

```diff
@@ -310,8 +311,8 @@ private void addArchiveNodeData(MaintainerExecutor maintainerExecutor, Container
      * @throws RuntimeException if exit code != 0
      */
     public String getHardwareDivergence() {
-        String configServers = environment.getConfigServerHosts().stream()
-                .map(configServer -> "http://" +  configServer + ":" + 4080)
+        String configServers = environment.getConfigServerUris().stream()
+                .map(URI::getHost)
```

Comment:
This is fine because SpecVerifier apparently appends own port and protocol if not set (https://github.com/vespa-engine/vespa/blob/master/node-maintainer/src/main/java/com/yahoo/vespa/hosted/node/verification/commons/HostURLGenerator.java)